src/components/index.js

```diff
@@ -4,9 +4,11 @@ import 'vuetify/dist/vuetify.min.css';
 import Authentication from './Authentication/';
 import DataBrowser from './DataBrowser.vue';
 import Upload from './Upload.vue';
+import UpsertFolder from './UpsertFolder.vue';
 
 export {
   Authentication,
   DataBrowser,
   Upload,
+  UpsertFolder,
```

Comment:
What do people think about introducing the Markdown and MarkdownEditor into the public API? I can see the argument both ways; it's not a Girder-centric component, however I could see downstreams that are already using this library wanting to make use of it to provide consistent markdown editing. I think I'm slightly in favor of adding them.

Reply:
Maybe we could put them in a directory called internal or something to indicate they are subject to change?